dpnp/dpnp_iface_types.py

```diff
@@ -59,8 +59,14 @@
     "inf",
     "int",
     "int_",
+    "int8",
+    "int16",
     "int32",
     "int64",
+    "uint8",
+    "uint16",
+    "uint32",
+    "uint64",
```

Comment:
It would be good to sort the names based on lexicographical order.

Reply:
Done